tests/test_connection.py

```diff
@@ -37,7 +37,7 @@ def test_cli_unknown_version(capsys):
     """
 
     matcher = re.compile(
-        r"^aqtinstall\(aqt\) v.* on Python 3.*\n"
+        r"^INFO    : aqtinstall\(aqt\) v.* on Python 3.*\n"
```

Comment:
```suggestion
        r"[^\n]*aqtinstall\(aqt\) v.* on Python 3.*\n"
```
I think this change would allow the `INFO:` prefix, but not require it. This should make the test more resilient to differences in dev environment and potential changes to the format of the prefix.